superset/connectors/druid/models.py

```diff
@@ -102,9 +102,12 @@ class DruidCluster(Model, AuditMixinNullable, ImportMixin):
     broker_endpoint = Column(String(255), default='druid/v2')
     metadata_last_refreshed = Column(DateTime)
     cache_timeout = Column(Integer)
+    broker_user = Column(String(255))
+    broker_pass = Column(String(255))
```

Comment:
We cannot be storing this in clear... There's some logic to replicate in `superset.models.Database` around password.

Reply:
ok i will investigate that.

Reply:
is it sufficient if I just change the password to `password = Column(EncryptedType(String(255), config.get('SECRET_KEY')))`